Homework/Khang_Nguyen/is-happy.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @param {number} n
+ * @return {boolean}
+ */
+ var isHappy = function(n) {
+    const freq = { [n]: 1 }
```

Comment:
this should work but can you improve to O(1) space using Floyd algorithm. In a real interview, interviewer can ask you to improve time/ space complexity